stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -283,8 +286,8 @@ impl DBConfig {
         });
         match epoch_id {
             StacksEpochId::Epoch10 => true,
-            StacksEpochId::Epoch20 => (1..=11).contains(&version_u32),
-            StacksEpochId::Epoch2_05 => (2..=11).contains(&version_u32),
+            StacksEpochId::Epoch20 => (1..=12).contains(&version_u32),
+            StacksEpochId::Epoch2_05 => (2..=12).contains(&version_u32),
```

Comment:
Can we turn the maximum DB schema version into a constant, so we don't have to keep writing `=12`?